addons/project/models/project.py

```diff
@@ -759,6 +759,7 @@ def _track_template(self, changes):
         if 'stage_id' in changes and test_task.stage_id.mail_template_id:
             res['stage_id'] = (test_task.stage_id.mail_template_id, {
                 'auto_delete_message': True,
+                'parent_id': False,
```

Comment:
The fix is not correct as it erases the value, we could use ``clean_context`` directly in ``_message_track_post_template`` to avoid this issue on other records. 

I'm gonna open another task about it.